creating_diverse_inclusive_communities.adoc

```diff
@@ -1,32 +1,31 @@
-But if there's one thing that techies love more than meritocracy, it's data.
-And when data shows who contributes to the OS ecosystem, it paints a damning picture.
+This movement hit the tech sector especially hard. When Pinterest released a statement supporting the Black Lives Matter movement, two Black women who formerly worked there quickly called out their hypocrisy.
+
+At first glance, open source seems immune to the multi-pronged problems with diversity and inclusion plaguing digital technology companies.
+Open source ecosystems operate online; contributors work across countries, languages, and timezones to power projects that drive the web forward.
+In somec cases, people contribute purely out of love for these projects, not for free beer and stock.
+Given the prevalence of such altruistic intent, the thinking goes, open source is a true meritocracy, a space where all can thrive regardless of their pronouns.
+But if there's one thing that industry advocates love more than meritocracy, it's data.
+And when data show who contributes to the open source ecosystem, they paint a damning picture.
 
 This chapter offers an overview of open source communities' efforts to make projects more diverse and inclusive.
-It also offers advice open source community managers and project maintainers can enact to ensure their communities and projects benefit from a diverse contributor base.
+It also offers initial steps open source community managers and project maintainers can take to begin building communities and projects that benefit from a diverse contributor base.
 
 == The state of open source diversity
 
-A 2017 GitHub survey of 5,500 open source users and developers found that 95% of randomly selected respondents identified as male.
-Three percent identified as women, and 1% as non-binary.
-
-Now, consider that in 2019, the U.S. Bureau of Labor Statistics found that 28.7% of total employed Computer and Information Systems managers https://www.bls.gov/cps/cpsaat11.htm[were women].
+In 2019, the U.S. Bureau of Labor Statistics found that 28.7% of total employed managers in the "Computer and Information Systems" sector https://www.bls.gov/cps/cpsaat11.htm[were women].
 Of the total employed, 9.6% were Black or African American, 15.8% were Asian, and and 4.7% were Hispanic or Latino.
 
-As these surveys indicate, the tech sector is overwhelmingly White and male; open source communities are even more so.
-Far from being a meritocracy, open source projects are much less likely to see contributions from computer science employees of minority status.
-
+As these surveys indicate, the digital technology industry is overwhelmingly White and male; open source communities are even more so.
+A 2017 GitHub survey of 5,500 open source users and developers found that 95% of randomly selected respondents identified as male.
```

Comment:
Can we link to or otherwise cite this survey? You reference it several times. I suspect readers may want to dive deeper.